### ISSUE TITLE
Plex for XBMC has thumbnails broken

### ISSUE THREAD

**Issue by maxpiva** [japanesemediamanager/plexplugin#2]
Investigate further...

![rsz_2ss_2014-07-19_at_071627](https://cloud.githubusercontent.com/assets/1164141/3982397/b7577f92-2876-11e4-8378-25a3c98a2ca8.jpg)

**Comment by koolkidsk** [comment]
Heya, that screenshot looks like it's from Plex, not XBMC. Also, the easiest way to get jmm working on XBMC at the moment might be just getting the PleXBMC XBMC plugin to work for the hacky myanime Plex channel. I messed around with it for almost an entire day and  put everything I found here https://forums.plex.tv/index.php/topic/106593-plexbmc-xbmc-add-on-to-connect-to-plex-media-server/?p=825039 It seems jmm gives the images to XBMC via proxy of PleXBMC and the myanime plugin for Plex but they're just not being displayed unless the images are coming from the images in the myanime plugin folder located in \AppData\Local\Plex Media Server\Plug-ins\MyAnime.bundle. Not sure if that's fixable on your part or the guy behind PleXBMC, or both, but might be a good start to check things out.
